### PR TITLE
User stories

### PR DESCRIPTION
**Pull request by pavlov200912**
User stories for the link shortening project.

### CODE REVIEW

File: user-stories.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Пользовательские истории:
+- Как пользователь, я хочу зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+- Как пользователь, я хочу получать короткую уникальную ссылку по длинному URL, чтобы было проще её использовать.
+- Как пользователь, я хочу использовать выданную мне короткую ссылку и переходить на соответствующий сайт без задержек.
```
Comment (NekoNe) [inline user-stories.md]: It is really a good point about latency. 
From manager's perspective I would be pleased to hear "without latency" but from engineering perspective I would ask: How am I supposed to do that?
Could you clarify what does it mean "without latency"?

File: user-stories.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Пользовательские истории:
+- Как пользователь, я хочу зарегистрироваться и получить свой аккаунт с уникальным id, чтобы меня можно было отличить от других пользователей.
+- Как пользователь, я хочу получать короткую уникальную ссылку по длинному URL, чтобы было проще её использовать.
```
Comment (NekoNe) [inline user-stories.md]: Does user make short link herself? Or your service should generate it randomly?

File: user-stories.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+- Как пользователь, я хочу использовать выданную мне короткую ссылку и переходить на соответствующий сайт без задержек.
+- Как зарегистрированный пользователь, я хочу иметь возможность просматривать уже созданные мною ранее сокращения для ссылок, чтобы использовать их повторно.
```
Comment (NekoNe) [inline user-stories.md]: We have two different interpretations of this story with Dmitri.
Interpretations:

1. User can see a list of all of her shot links, in order to be able to remember them.
2. User can assign new link to her old shot links

So, what does it mean?

File: user-stories.md (hunk 1 above)
Comment (NekoNe) [reply user-stories.md]: And as a user why do I want to follow shot link without latency?

File: user-stories.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+- Как зарегистрированный пользователь, я хочу иметь возможность удалять созданные мною ранее сокращения для ссылок.
```
Comment (NekoNe) [inline user-stories.md]: You missed motivational part in this story.

Should you specify the behavior of a removed link?

File: user-stories.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+- Как зарегистрированный пользователь, я хочу, чтобы действия от моего аккаунта мог выполнять исключительно я, чтобы никто не смог изменить мои ссылки.
```
Comment (NekoNe) [inline user-stories.md]: I'll leave a comment in Slack.

File: user-stories.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+- Как зарегистрированных пользователь, я хочу чтобы созданные мною ранее ссылки и их полные версии видел исключительно я, чтобы обеспечить приватность.
```
Comment (NekoNe) [inline user-stories.md]: If I place my short link in some social media, how does your service will protect my short link?
Do you mean that only I as a user could follow my short links?

**Review by NekoNe — CHANGES_REQUESTED** [review]

File: user-stories.md (hunk 2 above)
Comment (pavlov200912) [reply user-stories.md:3]: Who is "herself"?

**Comment by pavlov200912** [comment]
@NekoNe fixed!

**Review by M0nsteR — APPROVED** [review]
Good work,
ACK

**Review by NekoNe — APPROVED** [review]

**Pull request merged**